### PR TITLE
Adding Classes User and Measure

### PR DESCRIPTION
**Pull request by luciagirasoles**

### CODE REVIEW

File: models/User.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+require 'json'
+require 'date'
+require_relative 'Measure.rb'
+
+
+class User
```
Comment (condef5) [inline models/User.rb]: Where is the method save user?

File: models/User.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    self.new(
+      user["email"], 
+      user["name"], 
+      user["gender"], 
+      user["set_milestone"],
+      self.relation(user["measures"])
+    )
+  end
+
+  def self.by_last_week
```
Comment (diegotc86) [inline models/User.rb]: This method could be called "filter_by_time" or something like that, and receiving the time period as a parameters.
filter_by_time(7) would return the last week and filter_by_time(30) the last month.

File: models/User.rb [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        user["set_milestone"], 
+        self.relation(user["measures"])
+      )
+    end
+  end
+
+  def self.find(email)
+    users = self.read
+    user = users.detect{ |user| user["email"] ==  email}
+    self.new(
```
Comment (condef5) [inline models/User.rb]: This code is duplicated with line 34.

File: models/User.rb (hunk 1 above)
Comment (luciagirasoles) [reply models/User.rb]: This method is in process to be created.

File: models/User.rb (hunk 2 above)
Comment (luciagirasoles) [reply models/User.rb]: This method is going to be improved as is suggested. Currently, this file only has what is needed to be able to upload 'View_measures.rb' view

**Pull request merged**